pkgs/games/mindustry/default.nix

```diff
@@ -191,6 +191,6 @@ stdenv.mkDerivation rec {
     platforms = platforms.x86_64;
     # Hash mismatch on darwin:
     # https://github.com/NixOS/nixpkgs/pull/105590#issuecomment-737120293
-    broken = stdenv.isDarwin;
+    broken = true; #stdenv.isDarwin;
```

Comment:
```suggestion
    # does not work with any maintained java version
    broken = stdenv.isDarwin or true;
```

Reply:
I'd add a link to the upstream issue: https://github.com/Anuken/Mindustry/issues/5114

Reply:
Added both the commend incl. url and "||". Please tell me if this looks good to you now.

Reply:
Looks good! :+1: